server/metrics_database.h

```diff
@@ -39,18 +39,20 @@ class Metrics_Database {
 
 	std::string Prepare_All_Metrics_Body() { //changed return type from char* (used in server.cpp)
 		std::string msg;
-		for (this->it=this->mdb.begin(); this->it!=this->mdb.end(); ++it) {
-			msg.append(this->it->first);
+		std::map<std::string, std::string>::iterator it;
+		for (it=this->mdb.begin(); it!=this->mdb.end(); ++it) {
```

Comment:
The iterator can be declared inside the `for` loop and you can use `auto` keyword.